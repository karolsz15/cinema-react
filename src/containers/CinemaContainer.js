import React, { Component } from 'react';
import { Route, Switch } from 'react-router-dom';

import Homepage from '../components/Homepage/Homepage';
import Menu from '../components/Menu/Menu';

//testing
import Trailers from '../components/Trailers/Trailers';
import Contact from '../components/Contact/Contact';
import Booking from '../components/Booking/Booking';
import Error from '../components/Error/Error';;

class CinemaContainer extends Component {
    render () {
      return (
        <React.Fragment>
            <div class="full">
                <Menu />
                <Switch>
                    <Route path ="/" exact component={Homepage} />
                    <Route path ="/trailers" component={Trailers} />
                    <Route path ="/booking" component={Booking} />
                    <Route path ="/contact" component={Contact} />
                    <Route component={Error} />
                </Switch>
            </div>
        </React.Fragment>
        );
    };
};
  
export default CinemaContainer;