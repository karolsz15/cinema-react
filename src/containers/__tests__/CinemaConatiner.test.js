import React from 'react';
import { shallow } from 'enzyme';
import CinemaContainer from '../CinemaContainer';

let wrapper = shallow(<CinemaContainer />);

describe('CinemaContainer component', () => {
  it('should render the CinemaContainer correctly', () => {   
    expect(wrapper).toMatchSnapshot();
  });
});