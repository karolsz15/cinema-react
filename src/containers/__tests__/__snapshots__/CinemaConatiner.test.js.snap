// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CinemaContainer component should render the CinemaContainer correctly 1`] = `
<div
  className="full"
>
  <Menu />
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/trailers"
    />
    <Route
      component={[Function]}
      path="/booking"
    />
    <Route
      component={[Function]}
      path="/contact"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</div>
`;
